Assets/SEETests/TestLibGit2Sharp.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using NUnit.Framework;
+using LibGit2Sharp;
+using System.IO;
+using System.Text;
+using SEE.Utils.Paths;
+using DiffMatchPatch;
+
+namespace SEETests
```

Comment:
We abondoned the namespace `SEETests`. All test classes should now belong to the namespace of their class under test.